include/odp/api/spec/classification.h

```diff
@@ -719,7 +719,7 @@ int odp_cls_capability(odp_cls_capability_t *capability);
  * values.
  *
  * @param name         Name of the class-of-service or NULL. Maximum string
- *                     length is ODP_COS_NAME_LEN.
+ *                     length is ODP_COS_NAME_LEN, including the null character.
```

Comment:
It doesn't have to be there, but the reason why it's there is because "string length" usually doesn't include the null character. So I think it's prudent to remind the API reader in all of these places that the null character is included. I didn't change these in v2.